bin/build-stub

```diff
@@ -7,7 +7,7 @@ $license = str_replace("\n", "\n * ", trim($license));
 
 $autoload = <<<'EOS'
     Phar::mapPhar('psysh.phar');
-    require 'phar://psysh.phar/.box/check_requirements.php';
+    require 'phar://psysh.phar/.box/bin/check-requirements.php';
```

Comment:
was there a box version change that this happened in? should we bump our `composer.json` to reflect it?

Reply:
Yep there was one, but `.lock` is commited for Box so it's ok

Reply:
Until the next time the lock file is updated ;)

Reply:
Fair